pydantic/env_settings.py

```diff
@@ -47,8 +47,7 @@ def _build_values(
         _secrets_dir: Union[Path, str, None] = None,
     ) -> Dict[str, Any]:
         return deep_update(
-            deep_update(self._build_secrets(_secrets_dir), self._build_environ(_env_file, _env_file_encoding)),
-            init_kwargs,
+            self._build_secrets(_secrets_dir), self._build_environ(_env_file, _env_file_encoding), init_kwargs
```

Comment:
for better readability (and following PEP8, afaik), I'd split the parameters over multiple lines, one each.

Reply:
I ran the configured code formatter so I would assume it would handle this if it was a problem no? Maybe not? I will wait for a review and if a maintainer would like it re-formatted I will do so.

Reply:
this is great, in black we trust (mostly)